### PR TITLE
Adds in jackets to go with the uniforms i made

### PR DESCRIPTION
**Pull request by Kmc2000**
Here we go,

So you automatically start with these on, any jobs with other special stuff don't worry it's not removed, it will now spawn in your backpack such as the captain's armor

Screens:

Some are armored to make them an actual option to replace the normal stuff, like sec.

Additionally, admins if you want the XO to show his pride for Ian, there's a subtype of the XO'S coat called xo/silly, use at your own discretion

@Vivalas there you go

:cl: Kmc2000
add: Added crew jackets to go with uniforms, you automatically start with them on
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -162,3 +162,143 @@
...
+
+/obj/item/clothing/suit/toggle/service/xo/silly //meme coat for XO
+	name = "Xo's modified service uniform"
+	desc = "Ian's favey wavey snuggley wuggly coat!."
+	icon_state = "service_hopsilly"
+	item_state = "service_hopsilly" //TODO add in item states
+	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank/internals/emergency_oxygen,/obj/item/weapon/reagent_containers/glass/bottle,/obj/item/device/radio)
+
+/obj/item/clothing/suit/toggle/service/xo
+	name = "Xo's modified service uniform"
```
Comment (Vivalas) [inline code/modules/clothing/suits/jobs.dm]: "XO" please.

Comment (Kmc2000) [inline code/modules/clothing/suits/jobs.dm]: Sighs deeply and painfully

Comment (Vivalas) [inline code/modules/clothing/suits/jobs.dm]: Maintainer = Official Grammar Nazi

**Comment by Kmc2000** [comment]
👌👌👌👌👌👊👊👊👊👊👊👍👍👍👍👍💃💃💃💃💃

**Pull request merged**